### ISSUE TITLE
Repositories missing

### ISSUE THREAD

**Issue by bordieris** [stevebauman/maintenance#50]
Controllers use repositories i.e.: 
Asset Categories controller (https://github.com/stevebauman/maintenance/blob/master/app/Http/Controllers/Asset/CategoryController.php) use Asset category repository:
App\Repositories\Asset\CategoryRepository;

But repositories code is not there

**Comment by maarbu** [comment]
I don't see any responses to this and I'm seeing the same thing.  Did you get it figured out?

**Comment by bordieris** [comment]
No. And no answers. I think the project and its forks are not supported anymore.

**Comment by maarbu** [comment]
Thanks bordieris.  In other comments the author mentioned there were other priorities he had to give time to.
